src/resource/nonRdfData.ts

```diff
@@ -262,7 +292,7 @@ async function writeFile(
     ...defaultGetFileOptions,
     ...options,
   };
-  const headers = new Headers(config.init?.headers ?? {});
+  const headers = new CrossHeaders(config.init?.headers ?? {});
```

Comment:
In case cross-fetch's Headers conflict with native fetch, might be better to avoid it:

```suggestion
  const headers = flattenHeaders(config.init?.headers ?? {});
```

and then have the subsequent functions that take `headers` as an argument only accept a `Record<string, string>`.